superset/config.py

```diff
@@ -163,6 +163,7 @@
     'zh': {'flag': 'cn', 'name': 'Chinese'},
     'ja': {'flag': 'jp', 'name': 'Japanese'},
     'de': {'flag': 'de', 'name': 'German'},
+    'pt-BR': {'flag': 'pt-BR', 'name': 'Portuguese/Brazil'},
```

Comment:
Do you see the flag? Shouldn't this be something like?
```
    'pt-BR': {'flag': 'br', 'name': 'Brazilian Portuguese'}, 
```

Reply:
It really is better, thank you!